### PR TITLE
Modify color coding and thresholds to preferred values

### LINKED ISSUE #2: General color coding improvements of website
Update the color scheme of a few of the variables display on the website. 

Some of these is probably specific to our use case, so at some point these values may need to be broken out to a config file, etc.

### PR DESCRIPTION
**Pull request by pbitzer**
Change color coding.

Fixes #2

### CODE REVIEW

**Comment by CAM-Gerlach** [comment]
Needs the same fix as the other PR, 

```bash
git rebase --onto v0.2.x master color_code
```

Also, just FYI, if you want to link and resolve an issue with a PR, the trigger is `(fixe?|close|resolve)(s|ed)? #\d+`; addresses won't do it (unless you don't want it to close the issue? If so I apologize and please revert my tweak).

File: src/sindri/config/website.py [hunk 1]
```diff
@@ -411,16 +410,16 @@
...
         "plot_data": {
             "delta_period": "1H",
             "threshold_period": "24H",
             "threshold_type": "max",
-            "variable": "charge_state",
+            "variable": "led_state",
             },
         "plot_metadata": {
-            "plot_title": "Charge State",
+            "plot_title": "Battery LED State",
```
Comment (CAM-Gerlach) [inline src/sindri/config/website.py]:
```suggestion
            "plot_title": "LED State",
```

Space is very tight here and the LED shows more than just the battery status (AFAIK) so I suggest shortening this

File: src/sindri/config/website.py [hunk 2]
```diff
@@ -139,19 +139,18 @@
...
     "alarm": [[-0.5, 0.5, 500], ["red", "green", "lime", "red"]],
-    "led_state": [[-0.5, 2.5, 6.5, 8.5],
-                  ["red", "gray", "green", "yellow", "red"]],
+    "led_state": [[-0.5, 2.5, 6.5, 9.5, 10.5, 11.5],
+                  ["red", "gray", "green", "yellow", "orange", "red", "maroon"]],
     "power_out": [[0.1, 6, 12, 24], STANDARD_COLORS[::-1]],
-    "power_load": [[1, 6, 9, 10, 15.5, 18, 19, 20, 24],
-                   STANDARD_COLORS[::-1] + ["teal"] + STANDARD_COLORS[1:]],
+    "power_load": [[1, 5, 11, 22, 26, 28],
+    			   ["black", "orange", "yellow", "gray", "green", "orange", "red"]],
```
Comment (CAM-Gerlach) [inline src/sindri/config/website.py]:
```suggestion
    "power_load": [
        [1, 5, 11, 22, 26, 28],
        ["black", "orange", "yellow", "gray", "green", "orange", "red"],
        ], 
```

Fix indentation and erroneous mixed tabs and spaces

**Review by CAM-Gerlach — COMMENTED** [review]
Two minor issues, other than that LGTM aside from the Git issue

**Comment by pbitzer** [comment]
> Needs the same fix as the other PR,
> 
> ```shell
> git rebase --onto v0.2.x master color_code
> ```
> 
> Also, just FYI, if you want to link and resolve an issue with a PR, the trigger is `(fixe?|close|resolve)(s|ed)? #\d+`; addresses won't do it (unless you don't want it to close the issue? If so I apologize and please revert my tweak).

Should be good to go!

**Comment by CAM-Gerlach** [comment]
Hmm, looks like its wacked out even more now. Hold on, I'll take a crack at fixing it myself. You'll need to `git fetch origin && git reset --hard origin/color_code` on your branch to re-pull the fixed version, if you want it.

**Comment by CAM-Gerlach** [comment]
@pbitzer The above command worked for me locally in your branch; not sure what happened on your machine.

Also, just FYI, standard convention is to limit Git commit summary lines to 72 (ish) characters (some conventions are as low as 50), and separate additional descriptions by a blank line, which also avoids display/rendering abnormalities with Github, CLI and other tools that expect this. Since we had to rebase this anyway, I tweaked a couple of your commit messages to conform, as well as fixuped the trivial review fix commits with duplicate commit messages and gave them a proper description. Don't forget you can (and should :) use the batch apply feature to apply all related review suggestions at once, and make sure to write a useful commit message. Thanks!

**Review by CAM-Gerlach — APPROVED** [review]
LGTM now, thanks @pbitzer !

**Review by CAM-Gerlach — APPROVED** [review]
LGTM now, thanks @pbitzer ! (Somehow the first review didn't get fully picked up properly by the automation)

**Pull request merged**